code/modules/mob/living/carbon/xenomorph/hive_datum.dm

```diff
@@ -451,7 +451,7 @@
 
 
 /datum/hive_status/normal/check_ruler()
-	if(!SSticker?.mode || !(SSticker.mode.flags_round_type & MODE_XENO_RULER))
+	if(!(SSticker?.mode.flags_round_type & MODE_XENO_RULER))
```

Comment:
i believe SSticker doenst need the first ?. but ask tivi
```suggestion
	if(!(SSticker?.mode?.flags_round_type & MODE_XENO_RULER))
```

Reply:
The SSticker?.mode is everywhere almost, so i dunno. A mob always have flags_round though

Reply:
the one after mode? is neccessary as it had SSticker.mode before